src/Uno.UI/UI/Xaml/Style/Generic/Generic.xaml

```diff
@@ -218,38 +218,40 @@
 	</xamarin:Style>
 
 	<!--Default style for Windows.UI.Xaml.Controls.TextBox-->
+	<Thickness x:Key="TextBoxTopHeaderMargin">0,0,0,4</Thickness>
+
 	<Style x:Key="XamlDefaultTextBox"
-				   TargetType="TextBox">
+           TargetType="TextBox">
```

Comment:
much better but there's still some spaces infiltrating

Reply:
@davidjohnoliver Maybe I'm doing something wrong. How can I remove all space indentations from the file? Is the button in Visual Studio sufficient? I'll try this later today

Reply:
@davidjohnoliver This should be fixed completely now

Reply:
@duke7553 The contents of the `Grid` seem to've swapped positions with the `VisualStateManager`, inside the template. Could you put them back in the original order, the contents of the Grid below the VSM?

Reply:
@davidjohnoliver Corrected. Let's try again 😆